service/controller/chart/v1/key/key.go

```diff
@@ -31,6 +33,14 @@ func ConfigMapNamespace(customResource v1alpha1.Chart) string {
 	return customResource.Spec.Config.ConfigMap.Namespace
 }
 
+func CordonReason(customObject v1alpha1.Chart) string {
```

Comment:
```suggestion
func CordonReason(customResource v1alpha1.Chart) string {
```

This is the new stuff that will last a long time. So we should use the latest naming everywhere.